### PR TITLE
us#129630 Fix mashups after buttons refactoring

### PR DESCRIPTION
**Pull request by olyabedulina**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -56,14 +57,18 @@ tau
 
 
             var $button = $(_.flatten([
-                '<button class="tau-btn tau-extension-board-tooltip" id="mashup-quick-encoding" data-title="Visual encoding" alt="Visual encoding">',
+                '<button style="margin: 0" class="tau-btn i-role-board-tooltip tau-extension-board-tooltip" id="mashup-quick-encoding" data-title="Visual encoding" alt="Visual encoding">',
                     '<div class="tau-icon-colorpicker">',
                     '</div>',
-                '</button>'
+                '</button>',
```
Comment (victor-homyakov) [inline index.js]: Remove trailing space

File: index.js [hunk 2]
```diff
@@ -56,14 +57,18 @@ tau
 
 
             var $button = $(_.flatten([
-                '<button class="tau-btn tau-extension-board-tooltip" id="mashup-quick-encoding" data-title="Visual encoding" alt="Visual encoding">',
+                '<button style="margin: 0" class="tau-btn i-role-board-tooltip tau-extension-board-tooltip" id="mashup-quick-encoding" data-title="Visual encoding" alt="Visual encoding">',
```
Comment (victor-homyakov) [inline index.js]: Maybe move inline style into CSS?

File: index.js [hunk 3]
```diff
@@ -56,14 +57,18 @@ tau
...
                     '</div>',
-                '</button>'
+                '</button>',
                 ]).join(''));
 
             if (!$el.find('#mashup-quick-encoding').length) {
-                $el.find('[role=actions-button]').before($button);
+
+                var $elWrapper = $el.find('.tau-board-header__control--actions');
+                var $elAnchor = $elWrapper.length ? $elWrapper: $el.find('[role=actions-button]');
```
Comment (victor-homyakov) [inline index.js]: Code formatting: missing space before `:`

File: index.js (hunk 1 above)
Comment (olyabedulina) [inline index.js]: done

File: index.js (hunk 2 above)
Comment (olyabedulina) [inline index.js]: I need inline style for old and new code compability. After my css for tp is released I will remove inline styles.

File: index.js (hunk 3 above)
Comment (olyabedulina) [inline index.js]: fixed.

**Comment by victor-homyakov** [comment]
LGTM

**Pull request merged**